src/SWP/Bundle/WebhookBundle/composer.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "swp/webhook-bundle",
+  "type": "symfony-bundle",
+  "description": "Webhooks support for Symfony projects.",
+  "keywords": ["webhook bundle", "webhook", "publisher"],
+  "license":"AGPL-3.0",
+  "authors": [
+    {
+      "name": "Paweł Mikołajczuk",
+      "email": "pawel.mikolajczuk@sourcefabric.org"
+    }
+  ],
+  "repositories": [
+    {
+      "type": "path",
+      "url": "../../Component/*"
+    }
+  ],
+  "require": {
+    "php": "^7.0",
+    "swp/webhook": "^1.2"
```

Comment:
this bundle should require storage bundle

Reply:
yeah, requirements and tests are WIP